src/content/quickstarts/ccip-cross-chain-nft.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: "CCIP Cross-Chain NFT"
+description: "Mint an NFT on one blockchain from another blockchain using Chainlink CCIP."
+image: "QuickStarts-Batch-Collection-Reveal.webp"
+products: ["ccip"]
+time: "180 minutes"
+requires: "Wallet with gas token & ERC-677 LINK"
+---
+
+import { Accordion, Aside, CodeSample, ClickToZoom } from "@components"
+import { Tabs, TabsContent } from "@components/Tabs"
+
+## Overview
+
+This project demonstrates how to mint an NFT on one blockchain from another blockchain using Chainlink CCIP.
+
+<Aside type="caution" title="Disclaimer">
+  This tutorial represents an example of using a Chainlink product or service and is provided to help you understand how
+  to interact with Chainlink's systems and services so that you can integrate them into your own. This template is
+  provided "AS IS" and "AS AVAILABLE" without warranties of any kind, has not been audited, and may be missing key
+  checks or error handling to make the usage of the product more clear. Do not use the code in this example in a
+  production environment without completing your own audits and application of best practices. Neither Chainlink Labs,
+  the Chainlink Foundation, nor Chainlink node operators are responsible for unintended outputs that are generated due
+  to errors in code.
+</Aside>
+
+## Objective
+
+Imagine that you want to go to a conference that sells tickets as NFTs on one chain, but you have funds on some other chain. You will need to bridge your tokens to that chain where the NFT contract exists, to mint the NFT, and optionally bridge funds back.
+
+In this tutorial, you are building an cross-chain NFT minting system. This project aims to mint an NFT on the destination blockchain by sending the `to` address from the source blockchain. This tutorial uses a relatively simple example so you can understand a basic cross-chain message, but you can expand it to accept payment for minting on the source blockchain, add extra features, and more.
+
+The basic architecture diagram of what we want to accomplish looks like this:
+
+<ClickToZoom src="/images/quickstarts/cross-chain-nft/detailed-architecture-diagram.png" />
+
+
+## Before you begin
+
+Before you start this tutorial, install the required tools:
+
+- Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+  - Run `git --version` to check the installation. You should see an output similar to `git version x.x.x`.
+- Install [Nodejs](https://nodejs.org/en/) 16.0.0 or higher
+
+
+1. Verify installation by typing:
+
+```shell
+node -v
+```
+
+and
+
+```shell
+npm -v
+```
+
+1. Clone the repo and install the packages:
+
+```shell
+git clone https://github.com/smartcontractkit/ccip-cross-chain-nft.git && \
+cd ccip-cross-chain-nft && \
+npm install
+```
+
+1. Compile the contracts:
+
+```
+npx hardhat compile
+```
+
+### Configure environment variables
```

Comment:
These sections are showing up under the Before you begin section. Should this be part of its own independent section. I think we typically call this "Steps to implement".

Also, is this tutorial in a format that works with the collapsible sections like in the [Batch Reveal](https://docs.chain.link/quickstarts/batch-reveal) quickstart? Add the collapsible sections if possible, but it's not a requirement if they conflict with other elements on this page.